pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -146,6 +149,28 @@ func fixLibvirtEnabled() (bool, error) {
 	return true, nil
 }
 
+func checkLibvirtVersion() (bool, error) {
+	logging.Debugf("Checking if libvirt version is >=%s\n", minSupportedLibvirtVersion)
+	stdOut, stdErr, err := crcos.RunWithDefaultLocale("virsh", "-v")
+	if err != nil {
+		return false, fmt.Errorf("%v : %s", err, stdErr)
+	}
+	installedLibvirtVersion, err := semver.NewVersion(strings.TrimSpace(stdOut))
+	if err != nil {
+		return false, fmt.Errorf("Unable to parse installed libvirt version %v", err)
+	}
+	supportedLibvirtVersion, err := semver.NewVersion(minSupportedLibvirtVersion)
+	if err != nil {
+		return false, fmt.Errorf("Unable to parse %s libvirt version %v", minSupportedLibvirtVersion, err)
+	}
+
+	if installedLibvirtVersion.LessThan(supportedLibvirtVersion) {
+		return false, fmt.Errorf("You have %s version of libvirt installed but %s or higher required", installedLibvirtVersion.String(), minSupportedLibvirtVersion)
```

Comment:
"libvirt version %s is installed, but %s or higher is required" ?

Reply:
@cfergeau updated